bricks/virtualhub/mp_port.c

```diff
@@ -181,7 +119,7 @@ void pb_virtualhub_event_poll(void) {
 
     struct timespec timeout = {
         .tv_sec = 0,
-        .tv_nsec = 1000000,
```

Comment:
Why does this need to be faster? It should just be the 1ms tick.

Reply:
This entire section can go in principle. The clock_linux driver drives the clock tick now.

Reply:
100*1000:

![image](https://user-images.githubusercontent.com/12326241/221863684-46d08df4-d3e6-4d34-86ea-1580b24935f1.png)

1000*1000:

![image](https://user-images.githubusercontent.com/12326241/221863806-f0d85a37-7e38-4475-8094-8e8a1435fc1c.png)

Reply:
CPU overhead isn't too bad at all with 100*1000.

Reply:
Does 500*1000 work just as well?